explanation/gpus-instances.md

```diff
@@ -1,38 +1,47 @@
 (exp-gpus-instances)=
 # GPUs and instances
 
-Anbox Cloud has support for managing GPUs and can provide them to individual instances for rendering and video encoding functionality.
+Anbox Cloud supports managing GPUs and can provide them to individual instances for rendering and video encoding functionality.
 
 ```{important}
 This topic is applicable only for container instances because Anbox Cloud currently does not support GPU provisioning for virtual machines. This feature is planned for a future release.
 ```
 
-Anbox Cloud automatically detects GPU devices during the deployment and configures the cluster to use them. If no GPU is available, Anbox Cloud automatically falls back to the `null` platform that does not perform any rendering. See {ref}`sec-null-platform` for the `null` platform configuration. 
+Anbox Cloud automatically adjusts its behaviour depending on the availability of a GPU and the presence of the `--enable-graphics` flag when you launch an instance:
 
-However, you can enable software rendering and video encoding by launching your application with the `--enable-graphics` flag. This makes it possible to run entirely without a GPU and still use rendering.
-
-## Required GPU slots
+    amc launch --enable-graphics my-application
 
-GPUs have limited capacity that can be shared amongst multiple instances. To fine-tune how many instances can run on a given node, configure the number of available GPU slots on the node.
+The use of `--enable-graphics` flag when you launch an instance decides which [platform](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/supported-rendering-resources/#supported-platforms)(WebRTC or `null`) is used for rendering.
```

Comment:
Should we keep the explanation of the platform which is used in here? It's not really relevant to the user, right? We should much rather explain it from the perspective of what the outcome of specifying the flag is. With adding it we enable pixel rendering whereas without we run in a non-pixel rendering mode which has less CPU consumption but never produces any visual output

Reply:
This is a bit confusing for me as well, because the entire chapter is going to explain the deployment with GPU equipped and influence the underlying instance with GPU support or not with `gpu-slots`. We could mention the `graphics-headless` or visual output when the --enable-graphics flag is used or not, but mentioning the CPU here seems unnecessary.